src/server/game/World/World.cpp

```diff
@@ -1952,6 +1952,9 @@ void World::SetInitialWorldSettings()
     LOG_INFO("server.loading", "Load Channels...");
     ChannelMgr::LoadChannels();
 
+    LOG_INFO("server.loading", "Loading AntiDos opcode policies");
```

Comment:
shouldnt it be "AntiDDoS? o.o

Reply:
Technically speaking, no. The first "D" in DDoS is "Distributed", which these policies are per-session, thus not distributed.

Reply:
I see